extra-dev/packages/coq-fcsl-pcm/coq-fcsl-pcm.dev/opam

```diff
@@ -9,8 +9,8 @@ license: "Apache-2.0"
 build: [ make "-j%{jobs}%" ]
 install: [ make "install" ]
 depends: [
-  "coq" {>= "8.15"}
-  "coq-mathcomp-ssreflect" {>= "1.15.0" & < "2.0"}
+  "coq" { (>= "8.19" & < "8.21~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "2.2.0" & < "2.3~") | (= "dev") }
```

Comment:
```suggestion
  "coq" {>= "8.19"}
  "coq-mathcomp-ssreflect" {>= "2.2.0"}
```
@aleksnanevski do you really want to update this package every time a new MathComp or Coq version comes out? I think it's easier to just go with lower bounds for Git-based packages, to avoid doing updates often.

Reply:
Ok. I'm really just learning this, so I'll do whatever you advise.